site/_collections/algolia.js

```diff
@@ -100,6 +100,22 @@ module.exports = collections => {
       enumerable: true,
     });
 
+    // Add API types to algolia doc to allow fully qualified searches,
+    // like chrome.webRequest.onBeforeSendHeaders instead of just being
+    // able to search onBeforeSendHeaders which also appears in the content
+    if (item.data.api && item.data.chromeApiNamespaces[item.data.api]) {
+      const apiNamespace = item.data.chromeApiNamespaces[item.data.api];
+
+      const apiNamespacePaths = [];
```

Comment:
If someone changes signature of data.chromeApiNamespaces object, this code will break, but it will possibly not break the build, only make the search index not updating correctly (so will fail quietly). 
Can you extract this block to a separate function and add a test, so that tests break if this would to happen?

Reply:
Added a test, verifying correct paths are built for two exemplary APIs, `events` and `usb`.